Animal Competition/Java/Cheetah.java

```diff
@@ -20,19 +20,19 @@ public boolean hasTail() {
     return true;
   }
 
-  public boolean canClimb(){
-    return true;
+  public int canClimb(){
```

Comment:
Add some documentation as to what the values will mean; ie. is there a min/max value and what is a low value mean compared to a high value.  Also the name of the function should also be changed to reflect its true meaning.

Reply:
Don't forget about the naming of the function... `canClimb` is misleading as you'd think it's value would be yes or no.